zyte_spider_templates/spiders/ecommerce.py

```diff
@@ -55,6 +59,13 @@ class EcommerceSpiderParams(BaseSpiderParams):
                         "Use this when some subCategory links are misidentified by ML-extraction."
                     ),
                 },
+                EcommerceCrawlStrategy.direct_product: {
+                    "title": "Direct URLs to Product",
```

Comment:
We don’t support multiple URLs yet. We can either change it to “URL”, or wait until multiple URL input is available.

I’d probably prefer to wait until multiple URL input is available, because it doesn’t look very useful with a single URL. 

But there might be use cases, so no strong opinion.